test/integration/admin_login_test.rb

```diff
@@ -185,4 +185,32 @@ def setup
       assert page.has_content?("Order #{@order.id}")
     end
   end
+
+  test "an admin can retire an item" do
+    ApplicationController.any_instance.stubs(:current_user).returns(user)
+    Item.destroy_all
+    @item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200, retired: false)
```

Comment:
Line is too long. [110/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.